blueprints/ember-bootstrap/index.js

```diff
@@ -178,13 +180,24 @@ module.exports = {
     let file = 'ember-cli-build.js';
     let bootstrapVersion = this.bootstrapVersion;
     let preprocessor = this.preprocessor;
+    let config = this.retrieveExistingConfiguration();
     let settings = {
       bootstrapVersion
     };
 
-    settings.importBootstrapFont = (bootstrapVersion === 3);
+    if (bootstrapVersion === 4) {
+      settings.importBootstrapFont = false;
+    } else if (config.hasOwnProperty('importBootstrapFont')) {
+      settings.importBootstrapFont = config.importBootstrapFont;
+    } else {
+      settings.importBootstrapFont = true;
+    }
 
-    settings.importBootstrapCSS = (preprocessor === 'none');
+    if (config.hasOwnProperty('importBootstrapCSS')) {
```

Comment:
Hm, but sometime (maybe when Sass support was introduced), this changed in `index.js`, so right now even if `importBootstrapCSS` is true it will be ignored if there has been a preprocessor detected. See https://github.com/kaliber5/ember-bootstrap/blob/master/index.js#L49-L58.

So we should align the logic, either allow to import the static CSS even with a preprocessor setup (would require changes in `index.js`), or not (would require the proposed change here), right?